### ISSUE TITLE
Add accessibility labels to icons and input fields

### ISSUE THREAD

**Issue by albullington** [inaturalist/SeekReactNative#161]
Google Play gives suggestions on tappable icons which require accessibility labels. The following icons should have labels:

- [x] Footer menu icons
- [x] Seek logo & menu items -> SideMenu
- [ ] GetStarted modal
- [x] LocationPicker back button and return to user location button
- [x] AR camera button, back button, and camera help button
- [x] SpeciesDetail back button
- [ ] Start now buttons in ChallengeScreen
- [ ] InputFields
- [x] GreenHeader text
- [ ] ChallengesCard text, Home
